Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroup.cpp

```diff
@@ -89,13 +100,30 @@ namespace AZ::SceneAPI::SceneData
             serializeContext->Class<DataTypes::IPrefabGroup, DataTypes::ISceneNodeGroup>()
                 ->Version(1);
 
+            serializeContext->Class<ProceduralMeshGroupRule, DataTypes::IRule>()
+                ->Version(1);
+
             serializeContext->Class<PrefabGroup, DataTypes::IPrefabGroup>()
-                ->Version(1)
+                ->Version(2) // added createProceduralPrefab
                 ->Field("name", &PrefabGroup::m_name)
                 ->Field("nodeSelectionList", &PrefabGroup::m_nodeSelectionList)
                 ->Field("rules", &PrefabGroup::m_rules)
                 ->Field("id", &PrefabGroup::m_id)
-                ->Field("prefabDomData", &PrefabGroup::m_prefabDomData);
+                ->Field("prefabDomData", &PrefabGroup::m_prefabDomData)
+                ->Field("createProceduralPrefab", &PrefabGroup::m_createProceduralPrefab);
+
+            AZ::EditContext* editContext = serializeContext->GetEditContext();
+            if (editContext)
+            {
+                editContext->Class<PrefabGroup>("Prefab group", "Configure actor data exporting.")
+                    ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
+                        ->Attribute("AutoExpand", true)
+                        ->Attribute(AZ::Edit::Attributes::NameLabelOverride, "")
+                    ->DataElement(0,
+                        &PrefabGroup::m_createProceduralPrefab,
+                        "Create default procedural prefab?",
+                        "If this flag is true then the default procedural prefab will be generated.");
```

Comment:
Maybe expand this a little bit to explain what is contained in the default procedural prefab?  Some people might look at this option and not know what exactly that means